governance/roles.md

```diff
@@ -283,4 +289,37 @@ Some ongoing projects may have teams where members are identified for
 additional roles and may be required to have specific expertise. For visibility,
 these additional project roles are listed below:
 
-* [Security Reviewers](../assessments/guide/security-reviewer.md/)
+## Lead rotations
+
+For certain facilitator roles, teams and projects that are ongoing, leadership
+rotations should be carried out every 6 months. This is to:
+
+* Ensure leadership continuity of projects and passing on of tribal knowledge
+* Ensure diverse perspective/breadth of knowledge
+* Prevention of commitment bias in projects
+* Provide more leadership opportunities for more community members
+
+A facilitator role/project lead position should be a candidate for rotation if the
+project/facilitation task:
+
+* Exists for more than 6 months
+* Involves more than 1 person
+
+### Rotation process
+
+The rotation process for should start at least 2 weeks before a rotation is due.
```

Comment:
Either "for" is un-needed here, or another word's missing...

Reply:
```suggestion
The rotation process should start at least 2 weeks before a rotation is due.
```